### PR TITLE
Design fixes for APM

### PR DESCRIPTION
**Pull request by cardosogabriel**

### CODE REVIEW

**Comment by objectiser** [comment]
@ammendonca Would you be able to review this?

**Comment by ammendonca** [comment]
Most of the fixes are valid, but using pf-select on Properties / Faults filtering on the sidebar is making it not work. I think I've faced such issue (not sure if I reported to angular-patternfly), thus I didn't used it.

image

Also,not sure the change from plus / minus signs on the left of selected properties / faults was on purpose or just from replacing other with chevron, as these have the purpose of showing they are being included / excluded.

image

was / should be

image

**Comment by objectiser** [comment]
Agree with Alex - the + / - should not be chevrons as they do indicate included/excluded.

**Comment by cardosogabriel** [comment]
@ammendonca , updated the PR. I tested and it works here. Can u check? Thanks

File: ui/src/main/scripts/plugins/directives/filter-sidebar/html/filter-sidebar.html [hunk 1]
```diff
@@ -44,94 +44,98 @@
...
-          ng-model="$root.testFilter">
-        </select-->
-        <div ng-show="selPropName">
-          <label> Value </label>
+        <ul ng-repeat="(type, value) in $root.sbFilter.criteria.properties | groupBy: 'name'" class="fa-ul">
+          <label class="sb-prop-title">{{ type }}</label>
+          <li class="sb-prop" ng-repeat="prop in value" ng-click="toggleExcludeInclude(prop)">
+            <span class="hk-cursor-pointer">
+              <i class="fa-li fa" aria-hidden="true"
+ -              ng-class="prop.operator === 'HASNOT' ? 'fa-minus-square sb-toggle-exc' : 'fa-plus-square sb-toggle-inc'"></i>
```
Comment (ammendonca) [inline ui/src/main/scripts/plugins/directives/filter-sidebar/html/filter-sidebar.html]: Is this `-` supposed to be here ?

File: ui/src/main/scripts/plugins/directives/filter-sidebar/html/filter-sidebar.html [hunk 2]
```diff
@@ -44,94 +44,98 @@
...
-          <span class="hk-cursor-pointer pull-right sb-prop-remove" ng-click="remFaultFromFilter(fault)">
-            <i class="fa fa-remove"></i>
-          </span>
-        </li>
-      </ul>
+        <ul ng-repeat="fault in $root.sbFilter.criteria.faults" class="fa-ul">
+          <li class="sb-prop" ng-click="toggleExcludeInclude(fault)">
+            <span class="hk-cursor-pointer">
+              <i class="fa-li fa" aria-hidden="true"
+ -              ng-class="fault.operator === 'HASNOT' ? 'fa-minus-square sb-toggle-exc' : 'fa-plus-square sb-toggle-inc'"></i>
```
Comment (ammendonca) [inline ui/src/main/scripts/plugins/directives/filter-sidebar/html/filter-sidebar.html]: Sam, probably a not wanted `-`.

**Comment by cardosogabriel** [comment]
@ammendonca , removed the errors and rebased. Good catch, I hadn't see them in the UI/code.

**Comment by objectiser** [comment]
@ammendonca ok to merge?

**Comment by ammendonca** [comment]
@objectiser Yes, LGTM.

**Comment by objectiser** [comment]
@cardosogabriel @ammendonca Thanks!

**Pull request merged**